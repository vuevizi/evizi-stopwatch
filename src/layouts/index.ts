import MainLayout from "./MainLayout/MainLayout";
import Header from "./Header/Header";
import Footer from "./Footer/Footer";
export {Header,MainLayout,Footer}