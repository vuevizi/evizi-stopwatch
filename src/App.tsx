import React from 'react';
import logo from './logo.svg';
import './App.css';
import {MainLayout} from "./layouts";
function App() {
  return (
    <div className="App">
        <MainLayout/>
    </div>
  );
}

export default App;
