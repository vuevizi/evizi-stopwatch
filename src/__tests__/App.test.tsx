import App from '../App';
import {shallow} from 'enzyme';
import {MainLayout} from '../layouts';
import React from 'react';
import toJson from 'enzyme-to-json';

describe('App Component', () => {
  const wrapper = shallow(<App />);
  const container = (
    <div className="App">
      <MainLayout />
    </div>
  );
  test('renders without crash', () => {
    shallow(<App />);
  });
  test('renders right container', () => {
    expect(wrapper.contains(container)).toEqual(true);
    expect(wrapper.find('div.App')).toHaveLength(1);
  });
  test('renders the Mainlayout', () => {
    expect(wrapper.contains(<MainLayout />)).toEqual(true);
  });
  it('renders correctly', () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
