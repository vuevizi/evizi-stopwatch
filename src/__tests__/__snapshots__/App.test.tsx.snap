// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<div
  className="App"
>
  <MainLayout />
</div>
`;
